### ISSUE TITLE
Tab completion error message "__docker_daemon_is_experimental: command not found"

### ISSUE THREAD

**Issue by mgoelzer** [docker/docker#30953]
**Description**

I'm hitting a bug with the bash completions in docker 1.13.1 on Ubuntu 16.10.  Typing `docker service <TAB>` results in `__docker_daemon_is_experimental: command not found`

**Steps to reproduce the issue:**
1.  Type `docker service ` and then press TAB
2.  You'll get output like this:

```
mike@docker:~$ docker service __docker_daemon_is_experimental: command not found
__docker_daemon_is_experimental: command not found

create   list     ps       rm       update   
inspect  ls       remove   scale    
```

**Describe the results you received:**

See above.

**Describe the results you expected:**

There should not be an error message.

**Additional information you deem important (e.g. issue happens only occasionally):**

I can suppress the error by commenting out line 2706 of `/usr/share/bash-completion/completions/docker` like this:

```
_docker_service() {
        local subcommands="
                create
                inspect
                ls list
                rm remove
                scale
                ps
                update
        "
        #__docker_daemon_is_experimental && subcommands+="logs"

        __docker_subcommands "$subcommands" && return
```

Now tab completion for `docker service` works correctly.

**Output of `docker version`:**

```
$ docker version
Client:
 Version:      1.13.1
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 06:50:14 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.13.1
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 06:50:14 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**Output of `docker info`:**

```
$ docker info
Containers: 2
 Running: 0
 Paused: 0
 Stopped: 2
Images: 1
Server Version: 1.13.1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 5
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: active
 NodeID: egu61acy7rhg5c50u6529lfk6
 Is Manager: true
 ClusterID: dixudoljrbwz3m9kydbvwfixf
 Managers: 1
 Nodes: 1
 Orchestration:
  Task History Retention Limit: 5
 Raft:
  Snapshot Interval: 10000
  Number of Old Snapshots to Retain: 0
  Heartbeat Tick: 1
  Election Tick: 3
 Dispatcher:
  Heartbeat Period: 5 seconds
 CA Configuration:
  Expiry Duration: 3 months
 Node Address: 10.0.2.15
 Manager Addresses:
  10.0.2.15:2377
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: aa8187dbd3b7ad67d8e5e3a15115d3eef43a7ed1
runc version: 9df8b306d01f59d3a8029be411de015b7304dd8f
init version: 949e6fa
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.4.0-31-generic
Operating System: Ubuntu 16.04.1 LTS
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 1.954 GiB
Name: docker
ID: XEHQ:PANC:IEQU:AITH:3L2K:LM6R:6VIK:PQQN:7A4O:PRR2:CXN4:6MYF
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Not relevant

ping @albers

**Comment by albers** [comment]
Thanks for reporting. There is already a fix for this in #30875.

**Comment by mgoelzer** [comment]
Thank you -- closing this one

**Comment by albers** [comment]
@mgoelzer The issue will be fixed in a potential 1.13.2 release.
See https://github.com/docker/docker/pull/30875#issuecomment-279323995 for a link to the fixed bash completion script.
